<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportFromOpenIHM</class>
 <widget class="QDialog" name="ImportFromOpenIHM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Project from OpenIHM file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Database File:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtFilename">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="cmdGetDB">
        <property name="toolTip">
         <string>Select Database</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../../../../Brown/.designer/resources/images/addreport.png</normaloff>../../../../../../../Brown/.designer/resources/images/addreport.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Project Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblProjectName">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblStartDate">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Currency</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblCurrency">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="cmdImport">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="cmdClose">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdGetDB</sender>
   <signal>clicked()</signal>
   <receiver>ImportFromOpenIHM</receiver>
   <slot>getDB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdImport</sender>
   <signal>clicked()</signal>
   <receiver>ImportFromOpenIHM</receiver>
   <slot>importProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdClose</sender>
   <signal>clicked()</signal>
   <receiver>ImportFromOpenIHM</receiver>
   <slot>closeForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getDB()</slot>
  <slot>importProject()</slot>
  <slot>closeForm()</slot>
 </slots>
</ui>
